ide/libs.c.kohlschutter.junixsocket/external/junixsocket-common-2.2.0-license.txt

```diff
@@ -0,0 +1,210 @@
+Name: junixsocket
+License: Apache-2.0
+OSR: 4300
+Description: junixsocket is a Java/JNI library that allows the use of Unix Domain Sockets (AF_UNIX sockets) from Java.
+Origin: https://github.com/kohlschutter/junixsocket/
+Version: 2.2.0
```

Comment:
There is no need for three copies of the license file. Please remove two copies and rename the remaining copy to "junixsocket-license.txt". Add a new header line that references the three files:
```
Files: junixsocket-common-2.2.0.jar, junixsocket-core-2.2.0.jar, junixsocket-native-common-2.2.0.jar
```

What is more, the ALv2 requires to carry the NOTICE if the source package has one. In this case the original author failed to include it in his/her binaries, but we are good citizens of the ecosystem.

Here is the notice file:
https://github.com/kohlschutter/junixsocket/blob/master/NOTICE

Please add a file "junixsocket-notice.txt", with the contents of the above referenced file. The contents of that file will be picked up by the build system and integrated into the final notice file of NetBeans.

Reply:
licenses/notice changed/updated

Reply:
```
Some license files have incorrect headers
ide/libs.c.kohlschutter.junixsocket/external/junixsocket-license.txt does not contain the version 2.2.0 in its name
```
and if I add version to this, than it states, that there are 3 files and only one license, and so on...